### PR TITLE
New pull request as per requests from live-lab team for CW24 HOL

### PR DESCRIPTION
**Pull request by rghosh9**
Require a new PR for emergency PR submission for CW24 as per requests from live-lab team
Also, all minor formatting issues are resolved and live-lab retested by the Oracle Health team

Old PR here - https://github.com/oracle-livelabs/oracle-health/pull/7/files

### CODE REVIEW

File: compliance-form-generation-with-OCI-GenAI/other-livelabs/other-livelabs.md
Comment (ramonamagadan18) [inline compliance-form-generation-with-OCI-GenAI/other-livelabs/other-livelabs.md:1]: Hi @rghosh9 Please use the URLs from the new web domain: the LiveLabs webpage to now start with livelabs.oracle.com . For example for your first link will be: https://livelabs.oracle.com/pls/apex/f?p=133:180:115467694451312::::wid:3762

File: compliance-form-generation-with-OCI-GenAI/workshops/tenancy/manifest.json
Comment (ramonamagadan18) [inline compliance-form-generation-with-OCI-GenAI/workshops/tenancy/manifest.json:1]: Hi @rghosh9 ! Please use one single format for your labs: you had used "Lab 1"; "Lab2" etc ; which is perfect but now you have "Lab-3", which is not a LiveLabs format.

**Review by ramonamagadan18 — CHANGES_REQUESTED** [review]
Hi @rghosh9 Please add your WMS ID and see my other comments. Also, you have conflicts that must be solved.

**Comment by klazarz** [comment]
@rghosh9 please resolve conflicts

File: compliance-form-generation-with-OCI-GenAI/workshops/tenancy/manifest.json
Comment (rghosh9) [reply compliance-form-generation-with-OCI-GenAI/workshops/tenancy/manifest.json:1]: Thank you. Fixed all with a single format.

**Comment by rghosh9** [comment]
> Hi @rghosh9 Please add your WMS ID and see my other comments. Also, you have conflicts that must be solved.

WMS IS 11685

**Comment by rghosh9** [comment]
Need help on resolving these conflicts? Not sure. Any insights you can share?

**Pull request closed without merging**